doc/modules/cross_validation.rst

```diff
@@ -290,6 +290,37 @@ both testing and training. Notice that the folds do not have exactly the same
 size due to the imbalance in the data.
 
 
+.. _homogeneous_time_series_cv:
+
+HomogeneousTimeSeriesCV
+-----------------------
+
+:class:`HomogeneousTimeSeriesCV` is a variation of *k-fold* which 
+returns first :math:`k` folds as train set and the :math:`(k+1)` th 
+fold as test set. Note that unlike standard cross-validation methods, 
+successive training sets are supersets of those that come before them.
+
+This class can be used to cross-validate homogeneous time series data,
+i.e., each sample in dataset is observed every fixed time interval.
```

Comment:
wording here could be clearer; "samples are observed at fixed time intervals" would do.